### PR TITLE
Feature/add cd

### PR DESCRIPTION
**Pull request by clayreimann**
Most of this is cribbed from Netflix's excellent [nebula-release-plugin](https://github.com/nebula-plugins/nebula-release-plugin)

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -97,45 +104,47 @@ jacocoTestReport {
...
+    releaseBranchPatterns = [/.*/]
 }
 
 // see https://github.com/bintray/gradle-bintray-plugin
 // and https://github.com/bintray/bintray-examples/blob/master/gradle-bintray-plugin-examples/publications-example/build.gradle
 bintray {
     user = System.getenv("BINTRAY_USER")
     key = System.getenv("BINTRAY_API_KEY")
-    publications = ["ElideTestingFramework"]
+    publications = ["nebula"]
```
Comment (DeathByTape) [inline build.gradle]: Why is the `nebula`?

File: build.gradle [hunk 2]
```diff
@@ -1,19 +1,26 @@
...
+    id "com.jfrog.bintray" version "1.6"
 }
 
 group = "com.yahoo.elide"
-version = "1.0.0-" + (new Date()).format("yyyyMMddHHmm")
-
 description = "Elide Security Testing Framework"
 
+contacts {
+    "clayreimann@yahoo-inc.com" {
```
Comment (DeathByTape) [inline build.gradle]: Maybe we should setup a list e-mail for this? Can the outside world e-mail our internal lists?

Comment (clayreimann) [inline build.gradle]: Unsure, but I don't have a problem putting my email down.

File: build.gradle (hunk 1 above)
Comment (clayreimann) [inline build.gradle]: It's configured that way by the `nebula-project` plugin

**Comment by DeathByTape** [comment]
:+1:

**Pull request merged**